Documentation/ApiOverview/Database/RestrictionBuilder/Index.rst

```diff
@@ -146,6 +146,11 @@ Restrictions
    records need to be handled after the SQL results are fetched, by overlaying the records with
    :php:`BackendUtility::getRecordWSOL()`, :php:`PageRepository->versionOL()` or `PlainDataResolver`.
 
+If a Restriction needs to be enforced as it should always be applied, a restrition could implement the interface `EnforceableQueryRestrictionInterface`. Is this interface implemented for a Restrition, please take care of the following behavior:
```

Comment:
> If a Restriction needs to be enforced as it should always be applied

I would shorten this to just "If a restriction needs to be enforced" or "If a restriction should always be applied". I think it is enough and makes for smoother reading.